pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -675,3 +678,46 @@ type DigitalOceanSDConfig struct {
 	// +optional
 	RefreshInterval *v1.Duration `json:"refreshInterval,omitempty"`
 }
+
+
+
+// Docker SD configurations allow retrieving scrape targets from Docker Engine hosts.
+// This SD discovers "containers" and will create a target for each network IP and port the container is configured to expose.
+type DockerSDConfig struct{
```

Comment:
I think `Host` (Address of the Docker daemon.)  field missed here? That should be a required field